include/frozen/set.h

```diff
@@ -171,6 +174,8 @@ template <class Key, class Compare> class set<Key, 0, Compare> {
       : less_than_{comp} {}
   constexpr set(std::initializer_list<Key> keys) : set{keys, Compare{}} {}
 
+  constexpr set& operator=(const set &other) = default;
```

Comment:
clang 13 won't compile without it:
> error: definition of implicit copy assignment operator for 'set<unsigned int, 1>' is deprecated because it has a user-declared copy constructor

Reply:
I'm not sure it makes to have an assignment operator for a set whose keys are immutable... Could you share the wall error log and/or the input?

Reply:
But it can be used at runtime, why not? But I don't mind writing `= delete`. In any case, this operator already existed, only implicitly. clang error says that since we defined the copy constructor ourselves, then we probably want to define the assignment operator ourselves (in this case, it is default, but if it was custom, this warning would indicate a real bug)

Reply:
mmmh, I can see how that would interact nicely with `constinit`, deal!